### PR TITLE
Code Review!!

### PR DESCRIPTION
**Pull request by IjayAbby**
A request for Code Review for the HTML & CSS Course on Embedding Images and Video.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+    justify-content:space-between;
+
+}
+.recommended-list {
+    padding: 1rem;
+    width: 100%;
+}
+.recommended-video {
+    display: flex;
+
```
Comment (bertil291utn) [inline css/styles.css]: ## Using the Same Rule Repetitively[1]

- [ ] Avoid repeat `display: flex;` rule many times. 
Try like so:
````
.flex{
 display: flex;
}
````
Then add in your HTML elements where you require

`<div class="flex">`

###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+    padding: 1rem;
+    width: 100%;
+}
+.recommended-video {
+    display: flex;
+        
+}
+.recommended-video {
+    padding: 1rem;
+}
```
Comment (bertil291utn) [inline css/styles.css:236]: You have `recommended-video` class twice.
- [ ] Please merge those ones

File: index.html [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+                            <a href="#"><i class="fas fa-thumbs-up"></i></a>238
+                            <a href="#"><i class="fas fa-thumbs-down"></i></a>4
+                            <a href="#"><i class="fas fa-share"></i></a>SHARE
+                            <a href="#"><img src="images/save.png" alt="save"></a>SAVE
+                        </div>
+                    </div>
+                    <hr>
+                    <section class="description">
+                        <img src="images/microverse_logo.png" alt="Microverse log" style="width: 50px;height:50px">
+                        <section class="about-uploader">
```
Comment (bertil291utn) [inline index.html]: ## Using Inline Styles[1]
 - [ ] Avoid using inline styles. Move all your styles to CSS files

###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

**Review by bertil291utn — CHANGES_REQUESTED** [review]
**Status:** Changes requested 🔴  :shipit:? Not yet.

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

But in order to accomplish as the actual page, you required to do these changes, listed below: 

##### Mandatory changes

- [ ] Students **are not required** to add GitHub Actions as linters are required starting from the second project, therefore please remove your `.github/workflows/linters.yml` directory
-[ ] Check inline comments.

image

##### Optional changes

- [ ] Try to use [`<footer>`](https://www.w3schools.com/TagS/tag_footer.asp) tag for the footer.

- [ ] You might use a better reset CSS file. Like [normalize](https://necolas.github.io/normalize.css/8.0.1/normalize.css) [1]

- [ ] Consider use hex color instead rgb.[1]



###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_



<hr>


Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: css/styles.css [hunk 4]
```diff
@@ -167,7 +173,6 @@ body {
     margin: 14px 0px;
 }
 .featured-details {
-    display: flex;
     justify-content: space-between;
```
Comment (bertil291utn) [inline css/styles.css]: ## Remove `display:flex;` rule
If you have already a class `.flex{display:flex;}` declared, please remove all **display:flex** rules in your entire CSS file

**Review by bertil291utn — CHANGES_REQUESTED** [review]
**Status:** Changes requested 🔴  :shipit:? Not yet.

# Hey guys, you've been doing a great job.

All your changes seem amazing, however, you need to be careful about your last change related with `display flex:`,  check inline comments.

<hr>

Guys work in the requested changes above and request another review. 
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: css/styles.css (hunk 4 above)
Comment (IjayAbby) [reply css/styles.css]: Refactored our CSS & Index.html file to fix the issue.

**Review by HADDADSOHAIB — APPROVED** [review]
### Status: Project Approved :+1: 

### Hi @IjayAbby and @ggotora :hand: 

### Great Job, you implemented all the required changes by the previous TSE, you can go and merge this branch into master. However, this is some optional requirements that you could do before you merge:
- [Optional] Use the `<nav></nav>` tag for the navbar.
- [Optional] The share icon should be a button.

**Pull request merged**